ndlarflow/h5_to_root_ndlarflow.py

```diff
@@ -86,44 +86,72 @@ def main(argv=None):
 
         eventsToRun=len(events)
 
+        # Get the array of the trigger type for every event in the file
+        if useData==True:
+            triggerIDsData=flow_out["charge/events","charge/ext_trigs",events["id"][:]]
+            triggerIDsAll=np.array(np.ma.getdata(triggerIDsData["iogroup"]),dtype='int32')
+
+            triggerIDs = np.array( np.broadcast_to( (1 << 31) - 1, shape=eventsToRun ) )
+            for i in range(len(triggerIDsAll)):
+                if np.sum(triggerIDsAll[i])==0:
+                    continue
+                if 5 in triggerIDsAll[i]:
+                    triggerIDs[i] = 5
+                else:
+                    triggerIDs[i] = triggerIDsAll[i][0]
+        else:
+            triggerIDs = np.zeros( eventsToRun, dtype='int32' )
```

Comment:
Does `charge/ext_trigs` also exist for MC? For CAFs, I think -1 is used to set the "All" trigger flag if nothing is available; see the comment in ND_CAFMaker's [IRecoBranchFiller](https://github.com/DUNE/ND_CAFMaker/blob/main/src/reco/IRecoBranchFiller.h#L66)

Reply:
I'll check one of the 2x2 sim files you pointed me to on Slack. I'd be fine setting it to -1 otherwise.

Reply:
Noting here for the record that in a 2x2 sim file pointed to me by John and in one of our single neutrino spills for NDLAr, this branch does exist and sees to be always `1`

Reply:
For MC, do we want to change `triggerIDs = np.zeros( eventsToRun, dtype='int32' )` to use `np.ones` instead?

Reply:
yeah we can either change that or just let it grab the 1s from the hdf5 and get rid of that block entirely?

Reply:
OK, please modify this to use the hdf5 info for the MC as well, which I gather simply means removing the if statement.